src/RepoUtil/project.json

```diff
@@ -0,0 +1,21 @@
+{
+  "dependencies": {
+    "Microsoft.NETCore.App": {
+      "version": "1.0.0-rc2-3002702",
+      "type": "platform"
+    },
+    "Newtonsoft.Json": "8.0.3",
+    "System.Collections.Immutable": "1.2.0"
+  },
+  "buildOptions": {
```

Comment:
Since we aren't building with the cli then you don't even need the buildoptions section. Only the dependencies are necessary.